pkg/kubelet/kuberuntime/labels.go

```diff
@@ -56,6 +60,7 @@ type annotatedPodSandboxInfo struct {
 
 type labeledContainerInfo struct {
 	ContainerName string
+	ContainerType string
```

Comment:
yup, Done!

Since the next PR will add a `Type` field to kubecontainer.ContainerStatus which should also be typed, I went ahead and moved the new ContainerType type and constants from kuberuntime to kubecontainer.